website/docs/guides/best-practices/how-we-style/4-how-we-style-our-jinja.md

```diff
@@ -0,0 +1,37 @@
+---
+title: How we style our Jinja
+id: 4-how-we-style-our-jinja
+---
+
+## Jinja style guide
+
+- 🫧 When using Jinja delimiters, use spaces on the inside of your delimiter, like `{{ this }}` instead of `{{this}}`
+- 🆕 Use newlines to visually indicate logical blocks of Jinja.
+- 4️⃣ Indent 4 spaces into a Jinja block to indicate visually that the code inside is wrapped by that block.
+- ❌ Don't worry about Jinja whitespace control, focus on your project code being readable. The time you save by not worrying about whitespace control will far outweigh the time you spend in your compiled code where it might not be perfect.
```

Comment:
YESSSSSSS

Reply:
Although I would maybe say 
```suggestion
- ❌ Don't worry (too much) about Jinja whitespace control, focus on your project code being readable. The time you save by not worrying about whitespace control will far outweigh the time you spend in your compiled code where it might not be perfect.
```

My official ~rant~ guidance lives at the bottom of a random dbt utils issue: https://github.com/dbt-labs/dbt-utils/issues/513#issuecomment-1063647229